src/ui/server/middleware/meta/index.test.ts

```diff
@@ -0,0 +1,45 @@
+import { mockNext, mockReq, mockRes } from '../../test-mocks';
+import { Request, Response } from '../../../types';
+import { meta } from '.';
+import { PRODUCT } from '../../content-strings';
+
+const req: Request = mockReq();
+const res: Response = mockRes();
+const next = mockNext;
+
+describe('meta-data middleware unit test cases', () => {
```

Comment:
Can we align with other unit tests, we just have the name of the path and no need to mention it's a test 😛  keep it simple

```js
describe('middleware/headers/meta' () => ...
```

Reply:
Have updated.